android/build.gradle

```diff
@@ -51,12 +55,15 @@ allprojects {
 }
 
 ext {
-    backpackVersion     = "8.0.0"
+    backpackVersion     = "11.0.2"
```

Comment:
The latest and used below `backpack-android` version is `11.0.3`.

Reply:
good catch, updated.